osbs/api.py

```diff
@@ -653,6 +653,20 @@ def _check_labels(self, repo_info):
         except KeyError:
             pass
 
+        try:
+            _, version_value = labels.get_name_and_value(Labels.LABEL_TYPE_VERSION)
+            if version_value:
+                wrong_chars = \
+                    [denied for denied in VERSION_LABEL_FORBIDDEN_CHARS if denied in version_value]
+
+                if wrong_chars:
+                    logger.error("version label '%s' contains not allowed chars : %s",
+                                 version_value, wrong_chars)
+                    raise OsbsValidationException("version label contains not allowed chars %s" %
+                                                  wrong_chars)
+        except KeyError:
+            pass
```

Comment:
Which line above could raise KeyError? Suggest to write a comment to tell why this error can be ignored.

Reply:
getting version label, added comment

Reply:
I really like:
```python
try:
except:
else:
```
for such cases, when try block is as small as possible

Reply:
done